app/scripts/lib/auth-errors.js

```diff
@@ -187,6 +187,10 @@ function (_, Errors) {
     INVALID_EMAIL: {
       errno: 1023,
       message: t('Valid email required')
+    },
+    FORCE_AUTH_EMAIL_REQUIRED: {
```

Comment:
I had to dig deep to think about the original intent of this message. If the relier did things correctly, a user should not see this message. This message is aimed at RP developers who are integrating with FxA. The message's intent is to signal "hey, you forgot something."

In #1982, we check for required OAuth parameters on startup and redirect to `/400.html` if a parameter is missing. The name of the missing parameter is displayed to the user. We could do the same here. See #1740 and #1972 for more background. 

Or, perhaps we could do something like display "Unexpected error" which would be aimed at users, and also a "See more..." link that opens an expanded information element that displays RP developer facing information.

A third alternative is to use the developer console. Right now our console is pretty noisy, if reduce the amount of noise we could give RP devs information they can use.

I want to give RP developers useful information when they forget something.